projects/plugins/jetpack/tools/webpack.config.search.js

```diff
@@ -105,6 +105,9 @@ module.exports = {
 			requestToExternal,
 			requestToHandle: defaultRequestToHandle,
 		} ),
+		new webpack.ProvidePlugin( {
```

Comment:
Just curious where is `h` used?

Reply:
Good catch, this code is actually no longer necessary. For context, `preact.h` corresponds to `react.createElement` (and stands for [hyperscript](https://jasonformat.com/wtf-is-jsx/)).